components/webgpu/lib.rs

```diff
@@ -67,6 +67,8 @@ pub enum WebGPURequest {
     BufferMapAsync {
         sender: IpcSender<WebGPUResponseResult>,
         buffer_id: id::BufferId,
+        device_id: id::DeviceId,
+        scope_id: Option<u64>,
```

Comment:
If we are sending the scope ID with every operation, maybe we should move it out of the num?
Also, consider changing it to `Option<NonZeroU64>` to reduce the size